Felienne/hedy
# [FEATURE] Adds language and keyword language admin user filtering

**Description**
In this PR we improve the admin functionality for user filtering. We add columns for the chosen language and keyword language as well as filtering options for both these categories. The input fields are dynamically shown / hidden depending on the selected filtering input. We also fix a small bug where the teacher checkbox would always be marked, independent of the user confirmation.

**Fixes**
This PR fixes #2227 and fixes #2231

**How to test**
Make sure your account is set as admin in the environment. Then navigate to `/admin/users` and notice the added columns to the overview table. Verify that some columns are hidden on default but can still be shown. Also verify that the filtering on language and keyword language are working.
